cypress/e2e/gui/a11y.cy.js

```diff
@@ -0,0 +1,43 @@
+describe('EngageSphere', () => {
+  beforeEach(() => {
+    cy.mockCustomers().as('getCustomer')
+
+    cy.visit('/')
+    cy.injectAxe()
+  })
+
+  it('It finds no a11y issues in light mode in the customer table', () => {
+    cy.checkA11y()
+  })
+
+  it('It finds no a11y issues in dark mode in the customer table', () => {
+    cy.get('[aria-label*="theme"]').click()
+
+    cy.checkA11y()
+  })
+
+  it('It finds on a11y issues with the messenger\'s bubble button in dark mode', () => {
+    cy.get('[aria-label*="theme"]').click()
+
+    cy.checkA11y('[aria-label*="messenger"]')
```

Comment:
Que tal verificar antes que o tema mudou?

Algo assim: `cy.get('[data-theme="dark"]').should('exist')`

E então fazer a validação de acessibilidade.